vnext/Microsoft.ReactNative/XamlHelper.idl

```diff
@@ -17,5 +17,8 @@ namespace Microsoft.ReactNative
 
     DOC_STRING("Returns a Color from @JSValueArgWriter.")
     static Windows.UI.Color ColorFrom(JSValueArgWriter valueProvider);
+
+    DOC_STRING("Returns the attached property where the react tag is stored on a XAML Dependency Object.")
+    static XAML_NAMESPACE.DependencyProperty ReactTagProperty { get; };
```

Comment:
XamlUIService has ElementFromReactTag, maybe we should add ReactTagFromElement?

Reply:
I didn't see that class, I'll move the new property there as it makes more sense there anyway.